### ISSUE TITLE
Players can have seemingly infinite things impaled into them

### ISSUE THREAD

**Issue by TheVekter** [tgstation/-tg-station#8010]
So, uh, Garithos linked this in chat earlier.

http://puu.sh/gdeoC/47754385b1.png

@RemieRichards I don't think this is intended behavior...

**Comment by WJohn** [comment]
https://www.youtube.com/watch?v=p5HSepKMcbQ&t=1m08s

**Comment by palpatine213** [comment]
Feature.

**Comment by mrpain666** [comment]
Oh jesus christ I need to update my local server tomorrow and fucking see this for myself.

**Comment by RemieRichards** [comment]
@TheVekter It's intentional. Believe me here.
I felt that since it takes a normal amount of objects to kill someone, the infinite object thing remains as just humour, to me atleast, that humour felt like it fit in SS13.

if needs be, I'll cap it at something like 20 total objects, you're dead by then anyway but most of the humour remains.

**Comment by Steelpoint** [comment]
How much damage does, for example, a metal rod deal?

I heard a rumour it deals around 20 damage on impalement, and if that's all there is to it then I don't need to explain how bad that is.

**Comment by RemieRichards** [comment]
it deals w_class*5 on impact, which for rods is 10?
However I've just nerfed it to w_class*4
20/15 would probably be the throw_force of the rods, which is only called if they DIDN'T get embedded, so the two damages never stack.

I've also made it impossible for rods(and 90% of embedding objects to not embed without being thrown by an explosion (the exception is throwing stars, which have their throw_speed already defined at the embed_throwspeed_threshold)

I've also unhardcoded pretty much every multiplier tied to embedded objects, for uniqueness and customisability.
